docs/usage.md

```diff
@@ -121,6 +121,12 @@ Specifies the directory to write output files.
 
 Default is `dc_generated/` within current working directory.
 
+### `-oa`, `--observation-about`
+
+Specificies whether the observationAbout value of StatVarObservation nodes should be checked against the Data Commons API for existence. 
```

Comment:
Specifies whether to perform existence checks for places found in StatVarObservation nodes.

Reply:
They are by large locations / places.  I was just suggesting that so it's more readable. You can include a more accurate description of course.